scraper/src/insert_plant_relations.js

```diff
@@ -0,0 +1,111 @@
+import pgPromise from 'pg-promise';
+import csv from 'csvtojson';
+
+import { config } from 'dotenv';
+config();
+const pgp = pgPromise({});
+
+const db = pgp(process.env.DATABASE_URL);
+const companionsFilePath = process.argv[2] || 'data/Companions.csv';
+const antagonistFilePath = process.argv[3] || 'data/Antagonist.csv';
+
+/**
+ * Get Ids via the unique plant names in the database and populate a map 
+ *
+ * @param {*} map - a map with all unique plantnames and empty values
+ * 
+ * todo for later: user promise.all to requests all ids at the same time, and not wait for them 
```

Comment:
How is the current performance?

Reply:
For the current dataset, this part took 118.88ms when I timed it. It's just a part where I can see theoretical improvement.

Reply:
hehe, so I don't think we need that todo for now :wink: